### DISCUSSION TITLE
Has anyone run Mojo as an AWS Lamda function?

### DISCUSSION THREAD

**Discussion by rupertlssmith** [modular/modular#1342] [General]
You can write python AWS Lambdas, and I have a few currently deployed.

I know AWS has a feature they call Runtimes, which is a way of creating your own docker image for the runtime so that you can run AWS Lambdas on any language, not just the supported out of the box ones, such as python. Starting from an AWS base image for Linux, there ought to be little more to it that installing mojo into the image.

Has anyone done this already and have an image to share?

If not, I will try it out and post back with my findings.

**Comment by manatlan** [comment]
I install & run mojo every night in an "github's action" (ci/cd) (to run mojo scripts, for tests purpose)
But not on aws yet

**Comment by andreif** [comment]
Hey, did you succeed in your search?

**Comment by navroopbath** [comment]
Hey @rupertlssmith, that's a fantastic idea. Have you been able to try this and get a working image?

**Comment by albertocavalcante** [comment]
Just wondering if someone went further and gave it a try. Looking forward to finding a sample reference.

**Comment by carolinefrasca** [comment]
In an effort to centralize our community and ensure that all questions are answered as quickly as possible, GitHub Discussions for this repository will become read-only on September 26th and we ask that you share your questions, thoughts, and projects in [our Discord server](https://modul.ar/discord) instead.

Thank you for being part of our community and see you in Discord!
